src/modules/formlyBootstrap.js

```diff
@@ -70,8 +72,19 @@ angular.module('formlyBootstrap', ['formly'], function configFormlyVanilla(forml
     }
   });
 
+
   function getFieldTemplateUrl(name) {
     return 'fields/formly-field-' + name + '.html';
   }
 
 });
+
+angular.module('formlyBootstrap').run(function (formlyConfig, $http, $templateCache) {
+  formlyConfig.templateManipulators.preWrapper.push(function (template, options) {
+    if (options.type !== 'input' || (!options.templateOptions.addonLeft && !options.templateOptions.addonRight)) {
+      return template;
+    }
+    var tmpl = $templateCache.get('other/formly-other-bootstrap-addons.html');
```

Comment:
Done. So $http will check in cache before making the request? Goo to know, thanks!
And by the way, do you automatically get notified when I commit changes?